src/main/resources/config/liquibase/changelog/00000000000000_initial_schema.xml

```diff
@@ -88,50 +88,38 @@
                                  constraintName="fk_user_id"
                                  referencedColumnNames="id"
                                  referencedTableName="radar_user"/>
-        <!--<loadData encoding="UTF-8"-->
-                  <!--file="config/liquibase/users-new.csv"-->
-                  <!--separator=";"-->
-                  <!--tableName="radar_user">-->
-            <!--<column name="activated" type="boolean"/>-->
-            <!--<column name="created_date" type="timestamp"/>-->
-        <!--</loadData>-->
-        <!--<dropDefaultValue tableName="radar_user" columnName="created_date" columnDataType="datetime"/>-->
 
         <loadData encoding="UTF-8"
                   file="config/liquibase/authorities.csv"
                   separator=";"
                   tableName="radar_authority"/>
 
-        <!--<loadData encoding="UTF-8"-->
-                  <!--file="config/liquibase/users_authorities.csv"-->
-                  <!--separator=";"-->
-                  <!--tableName="radar_user_authority"/>-->
         <createTable tableName="jhi_persistent_audit_event">
             <column name="event_id" type="bigint" autoIncrement="${autoIncrement}">
                 <constraints primaryKey="true" nullable="false"/>
             </column>
-            <column name="principal" type="varchar(50)">
+            <column name="principal" type="varchar(1024)">
```

Comment:
any reason to have such a big varchar for audit events?

Reply:
Ah that's a mistake, the `principal` is just the username which is also `varchar(50)`, will revert this line. Thanks!